compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/text/selection/TextFieldSelectionManager.uikit.kt

```diff
@@ -136,7 +136,10 @@ private fun calculateSelectionMagnifierCenterIOS(
     return Offset(centerX, centerY)
 }
 
-private const val HideThresholdDp = 36
+/**
+ * Bottom drag point below the last text baseline after that magnifier is dismissed
+ * */
+internal const val MagnifierPostTravelDp = 36
```

Comment:
Would be better to use Dp as a type: `MagnifierPostTravel = 36.dp`

Reply:
Is it worth to loose compiler inlining just to have more cannonical name for variable that is only going to be multiplied by float density?

Reply:
Done